### ISSUE TITLE
Turn pin on/off

### ISSUE THREAD

**Issue by bodovix** [raspberry-sharp/raspberry-sharp-io#107]
I've been experimenting with this library but haven't been able to find a direct turn pin on or off? we can toggle or blink. but these require being me to be aware of the pins current state before toggling. 
To achieve this I ended up using the extension methods below. 
Is there a better way to do this currently? or is this something that we would maybe need?
(couldn't get the 'code text 'thing formatting properly...)

public static class GwydsGreatExtensionMethods
    {
        public static void TurnOn(this GpioConnection gpioConn, ProcessorPin pin)
        {
            GpioConnectionDriver driver = new GpioConnectionDriver();
            if (driver.Read(pin) == false)
            {
                gpioConn[pin] = !gpioConn[pin];
                Console.WriteLine("Light Turned on");
            }
            else
            {
                //already on. no need to toggle
                Console.WriteLine("already on.");
            }
        }
        public static void TurnOff(this GpioConnection gpioConn, ProcessorPin pin)
        {
            GpioConnectionDriver driver = new GpioConnectionDriver();
            if (driver.Read(pin) == true)
            {
                gpioConn[pin] = !gpioConn[pin];
                Console.WriteLine("Light Turned off");
            }
            else
            {
                Console.WriteLine("already off.");
            }
        }
    }

**Comment by rgelb** [comment]
I think this project is dead.  Last commit happened over 2 years ago.

**Comment by bfsmithATL** [comment]
No, not dead, just sleepy. It’s just not receiving new feature activity, although it’s a bummer to see those outstanding pull requests that are so stale. Core functionality is stable, I use it regularly on about 10 different RasPi’s.

**Comment by bfsmithATL** [comment]
@bodovix You might try checking out JTrotta's fork of raspberry-sharp, it has been kept up to date nicely:

https://github.com/JTrotta/RaspberrySharp
